### DISCUSSION TITLE
Enhancement - make header and footer stretch to width of page

### DISCUSSION THREAD

**Discussion by Jegelewicz** [ArctosDB/arctos#7756] [Ideas]
On certain pages you get white space and you lose the header and footer. Possible to make sure the header and footer stretches across the page?

Edit Agents

![image](https://user-images.githubusercontent.com/5725767/217389048-48b1c264-5472-4157-882b-8af73ba16a8c.png)

Search results

![image](https://user-images.githubusercontent.com/5725767/217389394-54127f1b-f441-4f80-8585-0ed34ed7ecae.png)
![image](https://user-images.githubusercontent.com/5725767/217389568-ead7ad61-81ea-4201-a98b-a0b0d7177e59.png)

**Comment by dustymc** [comment]
Need more info - both should be 1 view width wide, I suspect you're changing something after that's established or have some crazy browser or ???????????????

**Comment by Jegelewicz** [comment]
@campmlc please add your screenshots.

I am using Firefox
![image](https://user-images.githubusercontent.com/5725767/217568124-c2ffdde2-2196-49e4-bc6b-c66a8850b0d9.png)

I think the issues is it is 1 view width wide when I first go to https://arctos.database.museum/search.cfm

but then, I customize search results to add columns and that extends beyond the original "view". Which is going to happen to almost everyone. Any way we can force the view width to adjust when columns are added?

**Comment by dustymc** [comment]
>Any way we can force the view width to adjust when columns are added?

Grab both sides of your screen and pull *really* hard??

I'm never 100% sure of anything when it comes to CSS, but it should be the size of the application window - https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Values_and_units. CSS definitely does not provide a mechanism by which I might resize your windows.

**Comment by Jegelewicz** [comment]
From #5625 

![Screenshot 2023-02-08 21 14 25](https://user-images.githubusercontent.com/14808196/217716333-b9e191e4-073c-4ffc-aa09-ac6f125e912f.png)

**Comment by dustymc** [comment]
Can't re-create, closing.

**Comment by campmlc** [comment]
Reopening - this is a consistent problem across all forms. See the last issue I posted about the loan encumbrances and associated screenshot.

**Comment by dustymc** [comment]
@campmlc this isn't the first time I've seen hints that something isn't right with your machine, and this seems well beyond hint: your machine clearly isn't following the most basic of CSS rules. You should get it checked out, and expect that things aren't going to work until you do.

FWIW here's that form on my machine all scrunched up

<img width="503" alt="Screenshot 2023-02-09 at 7 34 24 AM" src="https://user-images.githubusercontent.com/5720791/217860464-69ade59a-5338-402c-bcd5-c2e7dbf2dd72.png">


and 'normal'

<img width="1637" alt="Screenshot 2023-02-09 at 7 34 38 AM" src="https://user-images.githubusercontent.com/5720791/217860580-4f49ac4b-93a9-4e5c-afb7-4e0006c8f424.png">

and stretched out to several times the width of my physical screen

<img width="1689" alt="Screenshot 2023-02-09 at 7 37 10 AM" src="https://user-images.githubusercontent.com/5720791/217861103-cb96289f-b9e1-42f0-8dd0-0601b7f43c5b.png">

Whatever I do, the element does what it should and I can see no evidence that anything in Arctos isn't operating properly.

**Comment by Jegelewicz** [comment]
This isn't just @campmlc it happens to me too, so we should try to figure out why so that we can help people make it work. It would be useful to know if anyone else is experiencing the issue. Perhaps in our next UI meeting we get everyone to try to recreate it.

**Comment by dustymc** [comment]
>happens to me too

Oh, sorry, I thought you were sharing screenshots.

URLs are as always useful too - eg maybe there's something in the data for whatever mysterious loan was screenshotted and not in the one I checked.

Yes, useful to know who's seeing this and as much as possible about their environments (eg maybe ya'll both have the same extension/OS/infection/proximity to large magnets/????).

I'm FF10.0b8 (64-bit) on mac 13.1 (22C65) but I use lots of browsers and versions and even more emulators and have not seen this in any of them

https://handbook.arctosdb.org/how_to/How-to-Use-Issues-in-Arctos.html#issue-protips

**Comment by campmlc** [comment]
I have a brand new Dell laptop, with windows and chrome.

**Comment by Jegelewicz** [comment]
Only extensions I have are 

![image](https://user-images.githubusercontent.com/5725767/217882339-f066cdf2-d396-4574-8b18-643adb009f9a.png)

**Comment by Jegelewicz** [comment]
@dustymc have you added enough columns to your search results? Get some results and add like five or ten columns.

**Comment by campmlc** [comment]
Also see loan item review form with actual data in it. Same thing, and
those columns are default.

**Comment by Jegelewicz** [comment]
This? https://stackoverflow.com/questions/61393217/how-can-i-horizontally-expand-my-footer-div

**Comment by campmlc** [comment]
Had this happen again today after doing a search on all Rausch records with my default number of columns. Here are some screenshots of the search, including what happens when I am viewing the media column and try to go up to the top of the page. This is not a problem with my computer - it is a problem with the fact that the current Arctos header only stretches as far as the search panels, and the header for the search results is not fixed. 


![Screenshot 2023-02-27 13 46 01](https://user-images.githubusercontent.com/14808196/221682928-e0cd9b8f-384f-495b-9bc9-1c5d3b653316.png)



![Screenshot 2023-02-27 13 46 32](https://user-images.githubusercontent.com/14808196/221682956-91866d49-1155-48b9-bc4f-ae632f6f0ec4.png)


![Screenshot 2023-02-27 13 45 49](https://user-images.githubusercontent.com/14808196/221682965-24dfad9b-74a4-4f2e-9887-f5ae76c22322.png)
